[2022-10-29T01:19:27.536+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: ENG_hourly_process_v1.get_hourly_demand scheduled__2022-10-29T00:05:00+00:00 [queued]>
[2022-10-29T01:19:27.548+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: ENG_hourly_process_v1.get_hourly_demand scheduled__2022-10-29T00:05:00+00:00 [queued]>
[2022-10-29T01:19:27.548+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-10-29T01:19:27.548+0000] {taskinstance.py:1363} INFO - Starting attempt 4 of 4
[2022-10-29T01:19:27.548+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-10-29T01:19:27.571+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): get_hourly_demand> on 2022-10-29 00:05:00+00:00
[2022-10-29T01:19:27.580+0000] {standard_task_runner.py:55} INFO - Started process 2898 to run task
[2022-10-29T01:19:27.583+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'ENG_hourly_process_v1', 'get_hourly_demand', 'scheduled__2022-10-29T00:05:00+00:00', '--job-id', '151', '--raw', '--subdir', 'DAGS_FOLDER/ENG_hourly_process_dag.py', '--cfg-path', '/tmp/tmpagx19g1m']
[2022-10-29T01:19:27.585+0000] {standard_task_runner.py:83} INFO - Job 151: Subtask get_hourly_demand
[2022-10-29T01:19:27.675+0000] {task_command.py:376} INFO - Running <TaskInstance: ENG_hourly_process_v1.get_hourly_demand scheduled__2022-10-29T00:05:00+00:00 [running]> on host 6800cd8e39a3
[2022-10-29T01:19:27.796+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=colmo786@gmail.com
AIRFLOW_CTX_DAG_OWNER=energy
AIRFLOW_CTX_DAG_ID=ENG_hourly_process_v1
AIRFLOW_CTX_TASK_ID=get_hourly_demand
AIRFLOW_CTX_EXECUTION_DATE=2022-10-29T00:05:00+00:00
AIRFLOW_CTX_TRY_NUMBER=4
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-29T00:05:00+00:00
[2022-10-29T01:19:27.806+0000] {logging_mixin.py:120} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:297 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-10-29T01:19:27.817+0000] {logging_mixin.py:120} INFO - INFO Module build_postgres_cnxn: DB Connection to host local_pgdb Ok
[2022-10-29T01:19:27.922+0000] {logging_mixin.py:120} INFO - INFO api_request_to_pandas: API request executed Ok. Number of records returned: 0
[2022-10-29T01:19:27.923+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/ENG_hourly_process_dag.py", line 49, in get_hourly_demand
    , user=user, password=password, port=port, verbose=True)
  File "/opt/airflow/dags/energy.py", line 161, in _get_hourly_demand
    df_demand.fecha = pd.to_datetime(df_demand.fecha.astype(str).str[:19], format='%Y-%m-%d %H:%M:%S')
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/generic.py", line 5487, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'DataFrame' object has no attribute 'fecha'
[2022-10-29T01:19:27.934+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=ENG_hourly_process_v1, task_id=get_hourly_demand, execution_date=20221029T000500, start_date=20221029T011927, end_date=20221029T011927
[2022-10-29T01:19:27.949+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 151 for task get_hourly_demand ('DataFrame' object has no attribute 'fecha'; 2898)
[2022-10-29T01:19:27.996+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2022-10-29T01:19:28.033+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
